src/components/Popups/ReplyField/ReplyField.tsx

```diff
@@ -64,16 +167,74 @@ export default class ReplyField extends React.Component<Props, State> {
         setCursorPosition(editorState.getSelection().getFocusOffset());
     };
 
+    handleMention = (): void => {
+        const { onMention } = this.props;
+        const { activeMention } = this.state;
+
+        if (!activeMention) {
+            this.hidePopup();
+            return;
+        }
+
+        if (onMention) {
+            onMention(activeMention.mentionString);
+        }
+
+        this.showPopup();
+    };
+
+    showPopup = (): void => {
+        const { activeMention } = this.state;
+
+        if (!activeMention) {
+            return;
+        }
+
+        const selection = window.getSelection();
+        const range = selection?.focusNode ? selection?.getRangeAt(0) : null;
+        const textNode = range?.endContainer;
+
+        let virtualElement = null;
+        if (range && textNode) {
+            range.setStart(textNode, activeMention.start);
+            range.setEnd(textNode, activeMention.start);
+
+            const mentionRect = range.getClientRects()[0];
+
+            virtualElement = {
+                getBoundingClientRect: () => mentionRect,
+            };
+
+            // restore selection
+            range.setStart(textNode, activeMention.end);
+            range.setEnd(textNode, activeMention.end);
+        }
+
+        this.setState({
+            popupReference: virtualElement,
```

Comment:
Do we want to set `popupReference` to `null` in case `range && textNode` evaluates to false?

Reply:
> Do we want to set `popupReference` to `null` in case `range && textNode` evaluates to false?

It is set to null in the above line 197

Reply:
> Do we want to set `popupReference` to `null` in case `range && textNode` evaluates to false?

I think we want the popup list to disappear when not focus on @mentions text. But let me check with designer